pkg/bindings/connection.go

```diff
@@ -144,10 +150,19 @@ func tcpClient(_url *url.URL) Connection {
 	connection := Connection{
 		URI: _url,
 	}
+	dialer := net.Dial
+	if v, found := os.LookupEnv("CONTAINER_PROXY"); found {
+		proxyURL, _ := url.Parse(v)
```

Comment:
this is redundant with lines 127-131, only parse this here and return the error should be enough